app/src/main/java/de/danoeh/antennapod/ui/screen/feed/SingleFeedSortDialog.java

```diff
@@ -16,11 +16,12 @@ public static SingleFeedSortDialog newInstance(Feed feed) {
         bundle.putLong(ARG_FEED_ID, feed.getId());
         bundle.putBoolean(ARG_FEED_IS_LOCAL, feed.isLocalFeed());
         if (feed.getSortOrder() == null) {
-            bundle.putString(ARG_SORT_ORDER, String.valueOf(SortOrder.DATE_NEW_OLD.code));
+            bundle.putString(ARG_SORT_ORDER, null);
         } else {
             bundle.putString(ARG_SORT_ORDER, String.valueOf(feed.getSortOrder().code));
         }
         SingleFeedSortDialog dialog = new SingleFeedSortDialog();
+        dialog.setShowDefaultSortButton(true);
```

Comment:
Hmm, to me it feels a bit weird to have a class called `SingleFeedSortDialog` but still the base class contains the default button already. How about making `populateList` protected and overwriting it in `SingleFeedSortDialog` to add the default?